pySDC/implementations/problem_classes/LeakySuperconductor.py

```diff
@@ -182,18 +184,22 @@ def get_non_linear_Jacobian(u):
             G = u - factor * self.eval_f(u, t) - rhs
 
             res = np.linalg.norm(G, np.inf)
-            if res <= self.params.newton_tol or np.isnan(res):
+            if res <= self.params.newton_tol and _n > 0:
```

Comment:
Why the `_n`?

Reply:
I mean, why the `_` in the name?

Reply:
Flakeheaven makes you add underscores to iteration variables that don't appear inside the loop, but now it does appear in the loop, so the underscore should be removed, you're right!